extensions/common/azure/azure-blob-core/src/test/java/org/eclipse/edc/azure/blob/cache/AccountCacheImplTest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ *  Copyright (c) 2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.azure.blob.cache;
+
+import org.eclipse.edc.spi.security.Vault;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.eclipse.edc.azure.blob.utils.BlobStoreUtils.createEndpoint;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+
+class AccountCacheImplTest {
```

Comment:
please test the "cache" behavior (requesting 2 times the same client will return the same instance)